File: leet004.c
#include <stdio.h>
#include <stdlib.h>

double findMedianSortedArrays(int* nums1, int nums1Size, int* nums2, int nums2Size) {
    
}

int main ()
{
    int nums_a[2] = {1, 3};
    int nums_b[1] = {2};
    int len_a = 2;
    int len_b = 1;
    // int nums_a[2] = {1, 2};
    // int nums_b[2] = {3, 4};
    // int len_a = 2;
    // int len_b = 2;

    double result = findMedianSortedArrays(nums_a, len_a, nums_b, len_b);
    printf("Resultado: %d\n", result);
}

File: leet004.py
def findMedianSortedArrays (nums1, nums2):
    nums = sorted(nums1 + nums2)
    if len(nums)%2 == 0:
        print((nums[len(nums)//2] + nums[len(nums)//2 - 1])/2)
        return (nums[len(nums)//2] + nums[len(nums)//2 - 1])/2
    else:
        return nums[len(nums)//2]


# resultado_a = findMedianSortedArrays([1, 3], [2])
resultado_b = findMedianSortedArrays([1, 2], [3, 4])
# resultado_c = findMedianSortedArrays([1, 2, 3, 4, 5], [6, 7, 8, 9, 10])

# print("a ->", resultado_a)
print("b ->", resultado_b)
# print("c ->", resultado_c)